nrfconnect/sdk-nrf
# lib: bin: lwm2m_carrier: update to v3.3.0

- Updated to latest version

Add support for Event Log object:

- Add support for the event log object to the LwM2M Carrier library.
- Add ability to set the log data of the event log object to the Serial
  LTE Modem application.

Deactivated and deprecated AT&T:

- Removed AT&T from the conceptual documentation.
- New library now requires AT&T to be explicitly enabled.

Add support for Binary App Data Container:

- Add support for the Binary App Data Container object to the LwM2M
  Carrier library.
- Add basic integration for the object to the Serial LTE Modem
  application.

applications: asset_tracker_v2: handle LWM2M_CARRIER_EVENT_LTE_POWER_OFF

- Added handling for LWM2M_CARRIER_EVENT_LTE_POWER_OFF event from the
LwM2M Carrier library, to ensure the LTE link is brought down and the
modem module enters the disconnected state.

updated documentation

- Updated the FOTA message sequence chart
- Updated changelog
